src/pymongo_migrate/mongo_migrate.py

```diff
@@ -32,6 +32,13 @@ def _deserialize(data, cls):
     return cls(**data)
 
 
+def _measure_time(func, argument):
```

Comment:
A function that get the function to execute and an argument, call the function with the given argument and return the time of executions.

Reply:
make it a contextmanager - it will be much easier to use

two alternative implementations:
```import time
from typing import Optional


class MeasureTime:
    def __init__(self):
        self.start = None
        self.stop = None

    @staticmethod
    def time() -> float:
        return time.time()

    @property
    def elapsed(self) -> Optional[float]:
        if self.start is None:
            return None
        return (self.stop or self.time()) - self.start

    def __enter__(self):
        self.start = self.time()
        return self

    def __exit__(self, exc_type, exc_val, exc_tb):
        self.stop = self.time()

mt = MeasureTime()
with mt:
    func(args)
print(f"func call lasted {mt.elapsed}s")
```

```
import contextlib

@contextlib.contextmanager
def measure_time():
    start = time.time()
    def elapsed():
        return time.time() - start

    yield elapsed

with measure_time() as elapsed:
    func(args)
    print(f"func call lasted {elapsed()}s")
````

Reply:
Thanks,
That great, I chose the second implementation you proposed, it nicer.

Reply:
After some struggling with the second approach, moved to the first one (define a class).